client/src/document/ingredients/browser-compatibility-table/index.tsx

```diff
@@ -109,6 +117,13 @@ function FeatureListAccordion({
           index={i}
           activeCell={activeRow === i ? activeColumn : null}
           onToggleCell={([row, column]: [number, number]) => {
+            const cell = `${column}:${row}`;
+            if (!clickedCells.current.has(cell)) {
+              clickedCells.current.add(cell);
+              gleanClick(
```

Comment:
I think it would be really impactful to record more here.

Whether this is a feature with a note (and what type) or just expanding a green check mark.